src/collects/seashell/backend/dispatch.rkt

```diff
@@ -108,10 +112,21 @@
             (send-message connection `#hash((id . -4) (success . #t)
                                             (result . #hash((pid . ,pid) (test_name . ,test-name) (result . ,test-res)))))]
            [(list pid test-name "error" exit-code stderr)
+            ;; check for asan error message in "{project}/{question}/.asan.{pid}"
```

Comment:
That's fine.

On Fri, Apr 15, 2016, 10:34 AM Yu Li <notifications@github.com> wrote:

> In src/collects/seashell/backend/dispatch.rkt
> <https://github.com/cs136/seashell/pull/512#discussion_r59883392>:
>
> > @@ -108,10 +112,21 @@
> >              (send-message connection `#hash((id . -4) (success . #t)
> >                                              (result . #hash((pid . ,pid) (test_name . ,test-name) (result . ,test-res)))))]
> >             [(list pid test-name "error" exit-code stderr)
> > +            ;; check for asan error message in "{project}/{question}/.asan.{pid}"
>
> Alright I'll change that. Just let you know that the asan message still
> needs to be sent back to the frontend through the "type: done" api if user
> runs their code without tests, and the compilation is success. (in which
> case the only options are to send back through "type: done", or through the
> last "type: stderr")
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/cs136/seashell/pull/512/files/a3c9f9e1d686bbd01503cd73212432f9a87b788f#r59883392>
>